src/openforms/forms/admin/tasks.py

```diff
@@ -0,0 +1,74 @@
+import logging
+import tempfile
+from pathlib import Path
+from uuid import uuid4
+from zipfile import ZipFile
+
+from django.conf import settings
+from django.core.files import File
+from django.core.management import call_command
+from django.template.loader import render_to_string
+from django.urls import reverse
+from django.utils.translation import gettext_lazy as _
+
+from openforms.accounts.models import User
+from openforms.celery import app
+from openforms.emails.utils import send_mail_html
+from openforms.utils.urls import build_absolute_uri
+
+from ..models import Form
+from ..models.form import FormsExport
+from ..utils import export_form
+
+logger = logging.getLogger(__name__)
+
+
+@app.task
+def process_forms_export(forms_uuids: list, email: str, user_id: int) -> None:
```

Comment:
`email` should not be needed since we have `user_id` and associated `user.email`